INFO:torch.distributed.nn.jit.instantiator:Created a temporary directory at /tmp/tmp8p6s4kh0
INFO:torch.distributed.nn.jit.instantiator:Writing /tmp/tmp8p6s4kh0/_remote_module_non_scriptable.py
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:git.cmd:Popen(['git', 'version'], cwd=/home/yanmy/enrich_release, universal_newlines=False, shell=None, istream=None)
DEBUG:wandb.docker.auth:Trying paths: ['/home/yanmy/.docker/config.json', '/home/yanmy/.dockercfg']
DEBUG:wandb.docker.auth:No config file found
2023-12-24 15:59:18,209 - INFO - episode_0 Task: wdc-K=5_Random f1-score: 0.6318131256952169, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 39, 40, 469, 470, 295, 296, 133, 134, 281, 282]
INFO:my_logger:episode_0 Task: wdc-K=5_Random f1-score: 0.6318131256952169, path: [0, 1, 2, 3, 4, 5, 6, 7, 8, 39, 40, 469, 470, 295, 296, 133, 134, 281, 282]
DEBUG:filelock:Attempting to acquire lock 140455786920544 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 140455786920544 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 140455786920544 on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Lock 140455786920544 released on /home/yanmy/.triton/autotune/Fp16Matmul_2d_kernel.pickle.lock
DEBUG:filelock:Attempting to acquire lock 140455786920544 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 140455786920544 acquired on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Attempting to release lock 140455786920544 on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
DEBUG:filelock:Lock 140455786920544 released on /home/yanmy/.triton/autotune/Fp16Matmul_4d_kernel.pickle.lock
